### ISSUE TITLE
Apache licence not applied according to its instructions

### ISSUE THREAD

**Issue by RalphCorderoy** [pires/docker-elasticsearch-kubernetes#30]
Hi, https://github.com/pires/docker-elasticsearch-kubernetes/blob/master/LICENSE#L178 explains how to apply the licence. That hasn't happened and so the licensing is a bit up in the air for those that want to be pedantic about such things before reusing it. This also goes for companies where employees would like to use it but have to ensure they're not dropping their employer in it.

**Comment by pires** [comment]
Hey Ralph! This has been subject to debate [here](https://github.com/pires/docker-elasticsearch-kubernetes/issues/27).

Thanks for reporting nonetheless.

**Comment by RalphCorderoy** [comment]
Thanks for the pointer. I wonder how many walk away without bothering to raise the issue and find out about your "hereby confirm" statement. :-(

**Comment by pires** [comment]
True! Want to open a PR that applies the licensing properly on all files?
